src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  @extend %flex-between;
+  padding: 0 50px;
+  height: 60px;
+  background-attachment: #fff;
```

Comment:
maybe you mean background-color?
color is invalid value for background-attachment
https://developer.mozilla.org/en-US/docs/Web/CSS/background-attachment